### DISCUSSION TITLE
Typelevel jobs board?

### DISCUSSION THREAD

**Discussion by rossabaker** [typelevel/general#110] [Ideas]
There was a recent tweet about it being difficult to find Typelevel Scala jobs, which I believe is more a failure to promote than a lack of openings.  I discussed this with a new colleague.

Status quo:
- `#jobs` channels on Scala and Typelevel Discords
   - invisible to the silent majority who doesn't hang out on chat
- scala/jobs Gitter: 
  - few and dwindling subscribers
  - Google-crawled, but I've never happened across a listing this way
- Tweets
  - ephemeral and disorganized
- Stack Overflow
  - nobody is tagging them Typelevel
  - I haven't even thought about that until I saw a link at $WORK

Ideas:
- Add a jobs section to typelevel.org
  - markdown and git is a barrier to recruiters
  - admin bandwidth is a barrier to posting them
  - no tooling to expire them
  - conflict of interest if the admins are trying to hire the same people?
- Punt and consider it a tooling need in a new typelevel.org
  - not a short-term fix
  - still an admin and conflict of interest problem?
- Try to take over a tag on an extant job board in hopes of critical mass of jobs and candidates
  - less work for us
  - no chance of success without concentrated promotion

**Comment by SethTisue** [comment]
> scala/jobs Gitter: few and dwindling subscribers

note that the Scala Discord already has a #jobs channel, which will likely soon replace scala/job-board

there's also the monthly Reddit jobs threads, current one is: https://www.reddit.com/r/scala/comments/ovuqb7/who_is_hiring_monthly_rscala_job_postings_thread/

**Reply by rossabaker** [reply]
Thanks, Seth.  I had no idea about the Reddit thread.  I don't know whether that speaks poorly for it or me!

**Reply by zarthross** [reply]
I've been aware of the Reddit Jobs thread, but that covers any Scala position, and not just Typelevel.  It can be sometimes difficult to sift through and figure out which ones are Typelevel or Lightbend style positions.

I also see it as ephemeral, last months postings might already be taken, and some companies don't keep posting month to month.

I've also tried posting some job positions, but as far as I could tell it didn't drive that much engagement with the position. 

I think something more Typelevel sanctioned/managed/linked would have a bigger impact driving people to roles with companies that use the Typelevel stack, and thus grow the Typelevel community.
